test/AngularSpec.js

```diff
@@ -1412,7 +1409,7 @@ describe('angular', function() {
       });
       var log = [];
       var self = {};
-      forEach(obj, function(val, key, collection) {
+      forEach(obj, /* @this */ function(val, key, collection) {
```

Comment:
It's a leftover from before its been disabled.

On Mon, Aug 1, 2016 at 1:37 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In test/AngularSpec.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72963406>:
>
> > @@ -1412,7 +1409,7 @@ describe('angular', function() {
> >        });
> >        var log = [];
> >        var self = {};
> > -      forEach(obj, function(val, key, collection) {
> > +      forEach(obj, /* @this */ function(val, key, collection) {
>
> Is this necessry (considering no-invalid-this has been disabled)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73#r72963406>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnk80ND-UFAJLgdIT7H0ifhVFV0uLks5qbdpvgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski